docs/content/users/quickstart.md

```diff
@@ -607,6 +607,66 @@ The Laravel project type can be used for [Statamic](https://statamic.com/) like
     ddev launch /cp
     ```
 
+## Sulu
+
+```bash
+mkdir my-sulu-site && cd my-sulu-site
+ddev config --project-type=php --docroot=public
+ddev start
+ddev composer create sulu/skeleton
+```
+
+!!!tip "Multilingual"
+    To set up the administration interface for example in French alongside the default English, add the following snippet on top of `config/packages/sulu_admin.yaml` ([all available localizations](https://sulu.crowdin.com/sulusulu)):
+
+    ```bash
+    sulu_core:
+    locales:
+        en: English
+        fr: Francais
+    translations:
+        - en
+        - fr
+    ```
+
+    `ddev exec bin/console sulu:admin:download-language` downloads any set language. The only exceptions are English and German which are shipping with Sulu.
+
+The content management part of Sulu is built upon webspaces. Each webspace configures a content tree. Each content tree may contain translations for different locales. Create your default webspace configuration `mv config/webspaces/example.xml config/webspaces/my-sulu-site.xml` and adjust the values for `<name>` and `<key>` so that they are matching your project:
+
+```bash
+<?xml version="1.0" encoding="utf-8"?>
+<webspace xmlns="http://schemas.sulu.io/webspace/webspace"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://schemas.sulu.io/webspace/webspace http://schemas.sulu.io/webspace/webspace-1.1.xsd">
+    <!-- See: http://docs.sulu.io/en/latest/book/webspaces.html how to configure your webspace-->
+
+    <name>My Sulu CMS</name>
+    <key>my-sulu-cms</key>
+```
+
+!!!warning "Caution"
+    Changing the `<key>` for a webspace later on causes problems. It is recommended to decide on the value for the key before the database is build in the next step.
+
+The information for the database connection is set in the environment variable `DATABASE:URL`. For development, clone the `.env` file by `cp .env .env.local` and change the environment variable to:
+
+```bash
+DATABASE_URL="mysql://db:db@db:3306/db?serverVersion=8.0.32&charset=utf8mb4"
+```
```

Comment:
This section should go earlier I think. People need to know this at the beginning. Maybe we can give them a standard way to do this with an `echo` or something.

We don't want to put an explicit patch version of mysql in there, it will break often. If we used `--database=mysql:8.0` above that does the job.

```suggestion
```bash
DATABASE_URL="mysql://db:db@db:3306/db?charset=utf8mb4"
```
```

Reply:
in regards of the position. i've followed the order sulu was using which seemed clear and logical to me. first you download the skeleton then you get the optional instructions for multiilingual  and the command how to download those, then you set the name and key which is necessary same as multilingual before building your environment and in the last step you prepare the env file with the database credentials. then you build it. so moving the env file before the other step i am not sure about.